bundles/org.aposin.mergeprocessor/src/org/aposin/mergeprocessor/model/IMergeUnit.java

```diff
@@ -109,6 +109,13 @@ default boolean hasRenaming() {
 		return getRenameMapping().entrySet().stream()
 				.anyMatch(entry -> !Objects.equals(entry.getKey(), entry.getValue()));
 	}
+	
+	/**
+	 * Shows the changes of the merge unit.
+	 */
+	default void showChanges() {
```

Comment:
I used the default method to add the method without breaking down the implementing classes, since, I noticed that [SVNMergeUnit2](https://github.com/haghani-mehran/MergeProcessor/blob/68e518e5c86d620d2acfcf13cfc86dac916b6f0e/bundles/org.aposin.mergeprocessor/src/org/aposin/mergeprocessor/model/svn/SVNMergeUnit2.java#L41) class has also implemented the IMergeUnit interface.

Reply:
OK, I understand your point of view that you don't want to break the API. 
Good argument.